src/conversion/azure/azure_converter.ts

```diff
@@ -12,16 +12,11 @@ import {
   VirtualNetworkConverter,
 } from '..';
 
-class AzureConverterImpl {
-  private readonly converters: ConverterStore<AnyAzureObject>;
-  private readonly entityStore: EntityStore;
-  private readonly symbolStore: SymbolStore;
-  private readonly vnetConverter: VirtualNetworkConverter;
+class AzureUniverse {
```

Comment:
When I suggested that we create an Azure Universe, I meant that we should have an object of type `UniverseSpec`, like `firewallSpec` (in src/spec/firewall.ts), that hard codes symbols like `AzureLoadBalancer` and `tcp`. This `UniverseSpec` will be almost the same as `firewallSpec`. The main differences are in the casing of the ports and protocols and the addition of Azure-specific symbols like `AzureLoadBalancer`.

Even if you weren't going to go as far as creating a UniverseSpec, you can still replace the AzureUniverse class with a POJO literal. There is no need for a class with a constructor here.

Reply:
This looks to require a bit of duplication. I put a TODO and will come back to this.